scripts/static-styles/config.js

```diff
@@ -24,13 +24,13 @@ import {
   Checkbox,
   Hr,
   Image,
-  Input,
+  // Input,
```

Comment:
Why are we removing it from style extraction?

Reply:
Static style extraction uses [react-docgen](https://github.com/reactjs/react-docgen) to extract the prop types automatically, same as Storybook. react-docgen isn't compatible with TypeScript in our current setup, that's why we need to disable those components temporarily. 

Storybook 6 will fix this incompatibility. Once we upgrade, I'll revisit the static style extraction as well.